specification/includes/physical-properties.json

```diff
@@ -34,6 +34,12 @@
       "minimum": 1,
       "example": 5000,
       "description": "Weight in grams."
+    },
+    "volumetric_weight": {
+      "type": "number",
+      "minimum": 1,
+      "example": 4500,
+      "description": "Volumetric weight in grams."
```

Comment:
Why do we need the volumetric weight here? 

Not every carrier calculates the volumetric weight in the same way. 
We have composed our own general formula for volumetric weight. Is it necessary to retrieve this from the carrier?

Reply:
Because in the current sprint we want to implement DHL, which returns it:
![Screenshot from 2020-05-13 11-26-12](https://user-images.githubusercontent.com/4649537/81795799-ce35cd00-950c-11ea-99b9-1f4c2b567b98.png)

If we want to recalculate the price, we need to know it. We can also do this using the returned length+width+height, but we are not the only ones invoicing anymore. Maybe other brokers want the actual value, because they do not want to use our generic formula. I would rather return it and not use it, than updating everything once a broker starts complaining.

Reply:
I see. While I was writing my comment I also thought about the fact that maybe our generic formula doesn't work for our brokers. I guess you're right. It's easier to remove it afterwards than to implement it later. I'll merge.